### ISSUE TITLE
[iOS] Pivot's native style displays only a header

### ISSUE THREAD

**Issue by jcantin-nventive** [unoplatform/uno#4464]
## Current behavior

The native style for Pivot is displayed incorrectly - only one header is visible.

## Expected behavior

The native style for Pivot is displayed correctly - all the headers are visible as well as the content for the selected header.

## How to reproduce it (as minimally and precisely as possible)

Replace this line in the Uno samples' App.xaml.cs
`Uno.UI.FeatureConfiguration.Style.UseUWPDefaultStylesOverride[typeof(CommandBar)] = false;`
with
`Uno.UI.FeatureConfiguration.Style.UseUWPDefaultStyles = false;`

Launch the samples and open the simple Pivot sample.

UWP style:
![image](https://user-images.githubusercontent.com/43351943/98167237-bd13ba80-1eb6-11eb-911c-889adc8b0f90.png)

Native style:
![image](https://user-images.githubusercontent.com/43351943/98167301-d74d9880-1eb6-11eb-850a-ca94189a0719.png)

## Environment

Nuget Package:

- [x] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s): - samples on the master branch with commit 0c1963a5

Affected platform(s):

- [x] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: 16.8.0 Preview 4.0)
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
@davidjohnoliver does this issue ring a bell ? Looks like the native styles somehow impact this control (which is not supposed to use any native presenters).

**Comment by Xiaoy312** [comment]
related: https://github.com/unoplatform/nventive-private/issues/114

**Comment by Xiaoy312** [comment]
https://github.com/unoplatform/uno/blob/11569ed96872bfda959ef1975af190b477de8674/src%2FUno.UI%2FUI%2FXaml%2FStyle%2FGeneric%2FGeneric.Native.xaml#L375
`Grid.Row` doesnt work on native controls, so it is taking the full height and blocks the content behind it.
The `UITabBar` needs to be wrapped inside a `Border` with the `Grid.Row` instead.

**Comment by jeromelaban** [comment]
@davidjohnoliver We should probably prevent the addition of known attached properties in these cases ? Grid/Canvas, etc ?

**Comment by davidjohnoliver** [comment]
> @davidjohnoliver We should probably prevent the addition of known attached properties in these cases ? Grid/Canvas, etc ?

Yes, it would probably be best to either fail when applying these properties, or else propagate them to the managed wrapper so they work correctly.

**Comment by jeromelaban** [comment]
@jcantin-nventive is there a specific project that is impacted by this issue ? (without naming it)

**Comment by jcantin-nventive** [comment]
> @jcantin-nventive is there a specific project that is impacted by this issue ? (without naming it)

yes there is. I will message you further

**Comment by MartinZikmund** [comment]
This works correctly for Skia renderer targets
